app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -364,9 +377,17 @@ public void saveAlert(final models.ebean.Alert alert) {
          * {@inheritDoc}
          */
         @Override
-        public long getEtag() {
-            final SqlQuery sqlQuery = Ebean.createSqlQuery("SELECT CURRVAL('portal.alerts_etag_seq') AS etag;");
-            return sqlQuery.findUnique().getLong("etag");
+        public long getEtag(final Organization organization) {
+            final SqlQuery sqlQuery = Ebean.createSqlQuery("SELECT etag from portal.alerts_etags where organization = ?;");
```

Comment:
Hrm. I agree, it's not an attribute of the organization. Should we then have a single versions table? That contains organization, asset type, and version?